aspnetcore/security/authentication/add-user-data.md

```diff
@@ -129,15 +129,15 @@ Update the `IdentityUser` derived class with custom properties. If you named the
 
 ::: moniker range=">= aspnetcore-3.0"
 
-[!code-csharp[Main](add-user-data/sample-3.0/Areas/Identity/Data/WebApp1User.cs)]
+[!code-csharp[](add-user-data/samples/3.x/SampleApp/Areas/Identity/Data/WebApp1User.cs)]
 
 Properties decorated with the [PersonalData](/dotnet/api/microsoft.aspnetcore.identity.personaldataattribute?view=aspnetcore-3.0) attribute are:
 
 ::: moniker-end
 
 ::: moniker range="< aspnetcore-3.0"
 
-[!code-csharp[Main](add-user-data/sample-2.2/Areas/Identity/Data/WebApp1User.cs)]
+[!code-csharp[Main](add-user-data/samples/2.x/SampleApp2/Areas/Identity/Data/WebApp1User.cs)]
```

Comment:
Thanks @eslam-fakhry ... you didn't have to go to all that trouble. Greatly appreciated!

Rename the 2.x sample `SampleApp`.

When I was describing `SampleApp` and `SampleApp2`, I just meant if there were **_two_** samples for the same version (in the same version folder). If there's only one sample in each of the version folders, we name them both `SampleApp` (usually ... there's a little bit of play in it ... sometimes the sample folders are named something more specific so that the dev can get a sense of approximately what each one is about ... e.g., `ConsoleSample` and `WebAppSample`). Anyway ... this one can just be `SampleApp` since it's the only one in the *2.x* folder.

Reply:
Hi @guardrex, I got that but there were two versions of 2.x (2.1 and 2.2). I renamed them SampleApp1 and SampleApp2 respectively or do I just delete 2.1 version?

Reply:
Ah ... sorry ... I should have said something about that.

*Yes!* ...... 🔪 **_KILL OFF_** ☠️ that 2.1 sample. :smile: We (almost always) only document and provide samples for the last minor version of each major version.

For example, when 3.1 comes out, we'll update all sample apps in-place in *3.x* folders to 3.1.

Reply:
Ok, thanks.

Reply:
... and you can see how using a *3.x* folder really helps on the *topic side of things*. The **_topic's links_** to the sample for the 3.1 upgrade don't change unless there's a line highlight or the actual sample file lines change.

Some of the topics have dozens of links ... it's a real time-saver for example back when going from 2.0 to 2.1 to 2.2. Each successive change saved 💰.